lib/l10nUtil.js

```diff
@@ -40,7 +40,9 @@ const braveBookmarksBarStringsPartPath = path.resolve(path.join(srcDir, 'brave',
 
 // src/components/components_strings.grd and any of its parts files.
 const chromiumManagementStringsPartPath = path.resolve(path.join(srcDir, 'components', 'management_strings.grdp'))
-const braveManagementStringsPartPath = path.resolve(path.join(srcDir, 'brave', 'chromium_src', 'components', 'management_strings.grdp'))
+const braveManagementStringsPartPath = path.resolve(path.join(srcDir, 'brave', 'app', 'management_strings.grdp'))
```

Comment:
these files should stay in `components`, they don't belong or make sense in `app`